lib/ProductOpener/SiteQuality_off.pm

```diff
@@ -0,0 +1,144 @@
+﻿package ProductOpener::SiteQuality;
+
+######################################################################
```

Comment:
Header should probably match the other ProductOpener modules and contain license info.